roles/lib_openshift/src/lib/base.py

```diff
@@ -22,6 +22,29 @@ def __init__(self,
         self.verbose = verbose
         self.kubeconfig = Utils.create_tmpfile_copy(kubeconfig)
         self.all_namespaces = all_namespaces
+        self.oc_binary = self._locate_oc_binary()
+
+    @staticmethod
+    def _locate_oc_binary():
+        ''' Find and return oc binary file '''
+        # https://github.com/openshift/openshift-ansible/issues/3410
+        # oc can be in /usr/local/bin in some cases, but that may not
+        # be in $PATH due to ansible/sudo
+        paths = os.environ.get("PATH", os.defpath).split(os.pathsep) + ['/usr/local/bin', os.path.expanduser('~/bin')]
+        paths = list(set(paths))
+
+        oc_binary = 'oc'
+        if six.PY2:
```

Comment:
> Ideally, we'd be able to use a library, but there doesn't seem to be one that we can just use without some work.

If 4~6 lines of Python code is the implementation used for Py2, why can't we use that for Py3 as well? We don't need an extra dependency nor maintaining packages.